e2e/tests/noticeOfChange_test.js

```diff
@@ -1,19 +1,41 @@
 const config = require('../config.js');
 const mandatorySubmissionFields = require('../fixtures/caseData/mandatorySubmissionFields.json');
+const representedCase = require('../fixtures/caseData/representedCase.json');
 
 let caseId;
 
 Feature('Notice of change');
 
-BeforeSuite(async ({I}) => {
+Scenario('Private solicitor obtains access to an unrepresented case', async ({I, caseListPage, caseViewPage, noticeOfChangePage}) => {
   caseId = await I.submitNewCaseWithData(mandatorySubmissionFields);
+  await I.signIn(config.privateSolicitorOne);
+  I.navigateToCaseList();
+  caseListPage.searchForCasesWithId(caseId);
+  I.dontSeeCaseInSearchResult(caseId);
+
+  await userCompletesNoC(I, caseListPage, noticeOfChangePage);
+
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  assertRepresentative(I, config.privateSolicitorOne.email, 'External', config.privateSolicitorOne.email, 'Private solicitors');
 });
 
-Scenario('Private solicitor obtains case access through NoC', async ({I, caseListPage, noticeOfChangePage}) => {
+Scenario('Private solicitor replaces respondent solicitor on a represented case', async ({I, caseListPage, caseViewPage, noticeOfChangePage}) => {
+  caseId = await I.submitNewCaseWithData(representedCase);
+  await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId);
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  assertRepresentative(I, 'Tom', 'Jones', 'test@test.co.uk');
+
   await I.signIn(config.privateSolicitorOne);
   I.navigateToCaseList();
   caseListPage.searchForCasesWithId(caseId);
   I.dontSeeCaseInSearchResult(caseId);
+
+  await userCompletesNoC(I, caseListPage, noticeOfChangePage);
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  assertRepresentative(I, config.privateSolicitorOne.email, 'External', config.privateSolicitorOne.email, 'Private solicitors');
+});
+
+const userCompletesNoC = async (I, caseListPage, noticeOfChangePage) => {
```

Comment:
small point, but this might make more sense in the `noticeOfChange.page.js` to follow the page object pattern set by `caseList.page.js`, like:
```
  await noticeOfChangePage.userCompletesNoC(caseId, 'Swansea City Council', 'Joe', 'Bloggs');
```
```
/* noticeOfChange.page.js */

  async userCompletesNoC (caseReference, applicantName, firstName, lastName) {
    this.navigate();
    await this.enterCaseReference(caseReference);
    await I.retryUntilExists(() => I.click('Continue'), this.fields.applicantName);
    await this.enterApplicantName(applicantName);
    this.enterRespondentName(firstName, lastName);
    await I.retryUntilExists(() => I.click('Continue'), this.fields.confirmNoC);
    this.confirmNoticeOfChange();
    await I.retryUntilExists(() => I.click('Submit'), '.govuk-panel--confirmation');
    I.see('Notice of change successful');
    await I.retryUntilExists(() => I.click('View this case'), '.case-title');
  },
```